packages/@aws-cdk/aws-apigateway/lib/authorizers/lambda.ts

```diff
@@ -31,7 +27,7 @@ export interface TokenAuthorizerProps {
    * The request header mapping expression for the bearer token. This is typically passed as part of the header, in which case
    * this should be `method.request.header.Authorizer` where Authorizer is the header containing the bearer token.
    * @see https://docs.aws.amazon.com/apigateway/api-reference/link-relation/authorizer-create/#identitySource
-   * @default 'method.request.header.Authorization'
+   * @default 'method.request.header.Authorization' for TOKEN and 'Header' for REQUEST
```

Comment:
Quoting from - https://docs.aws.amazon.com/apigateway/api-reference/resource/authorizer/#identitySource - "For the REQUEST authorizer, this is required when authorization caching is enabled. The value is a comma-separated string of one or more mapping expressions of the specified request parameters."

It seems like the values of these and their function is different enough across Token and Request types that they should be modeled differently in the CDK.

I would move this out of `LambdaAuthorizerProps` and into `TokenAuthorizerProps` and define property of array type in `RequestAuthorizerProps` that gets converted into a comma-separated string.